clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleHttpLoadBalancerCachingAgent.groovy

```diff
@@ -363,6 +363,7 @@ class GoogleHttpLoadBalancerCachingAgent extends AbstractGoogleLoadBalancerCachi
         service.affinityCookieTtlSec = backendService.affinityCookieTtlSec
         service.enableCDN = backendService.enableCDN
         service.portName = backendService.portName ?: GoogleHttpLoadBalancingPolicy.HTTP_DEFAULT_PORT_NAME
+        service.connectionDrainingTimeoutSec = backendService.connectionDraining?.drainingTimeoutSec ?: 0
```

Comment:
The linked docs don't make it look like `0` is an allowed value here. Why not `null`? 

It won't make a difference for the caching agent, but `0` will be the populated value in the edit LB UI - and if someone tries to upsert an LB with that value, I'm guessing they'll run into a problem.

Reply:
Zero is an acceptable value for the connection draining timeout.